src/NHPersistenceCompatibilityTests/Common/NugetHelper.cs

```diff
@@ -0,0 +1,54 @@
+﻿using NuGet;
+using System.Collections.Generic;
+using System.Linq;
+using System;
+
+namespace Common
+{
+    public class NugetHelper
+    {
+        string packageSource;
+        string fallbackPackageSource;
+
+        public NugetHelper(string packageSource = "https://packages.nuget.org/api/v2", string fallbackPackageSource = "https://www.myget.org/F/particular/")
```

Comment:
There shouldn't be a third argument. The thinking is that we should test the packages that are available to our users, so that means Nuget or MyGet currently. Ideally we would have preferred for Nuget to be the only source (it's a more properly curated release list than MyGet) but due to the nature of the release churn with beta's we need to have a backup.

Summary: I don't foresee a third argument adding any value in the future. In fact: I think even having 2 arguments is a code smell that we're using to work around current shortcomings.

I do however agree about the values in code. I shall move them to config, re-push and ping again.